www/templates/html/SiteMaster/Core/Auditor/Site/Review/EditForm.tpl.php

```diff
@@ -0,0 +1,77 @@
+<?php
+use \SiteMaster\Core\Auditor\Site\Review;
+
+/**
+ * @var $context \SiteMaster\Core\Auditor\Site\Review\EditForm
+ */
+?>
+
+<form method="POST">
+    <ul>
+        <li>
+            <?php 
+            $date_scheduled = '';
+            if ($context->review) {
+                $date_scheduled = date('Y-m-d', strtotime($context->review->date_scheduled));
```

Comment:
fixed in https://github.com/mfairchild365/site_master/commit/514b62b599d58b30bf8bf731d73dd69dad21da1d